plugins/aws/resoto_plugin_aws/resource/route53.py

```diff
@@ -127,6 +127,13 @@ def delete_resource(self, client: AwsClient) -> bool:
         client.call(aws_service="route53", action="delete-hosted-zone", result_name=None, Id=self.id.rsplit("/", 1)[-1])
         return True
 
+    @classmethod
+    def called_mutator_apis(cls) -> List[AwsApiSpec]:
+        return [
+            AwsApiSpec("route53", "change-tags-for-resource"),
+            AwsApiSpec("route53", "delete-hosted-zone"),
+        ]
```

Comment:
Further up there should also be a [ChangeResourceRecordSets](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/route53.html#Route53.Client.change_resource_record_sets) for deleting an individual RRSet.